Makefile

```diff
@@ -110,3 +120,17 @@ else
 endif
 endif
 endif
+
+ci-setup:
+	@echo "Installing Go..."
```

Comment:
You're not really installing go here, right? :grin:

Reply:
Right. I'll say "Go tools"